notebooks/en/llm_judge_evaluating_ai_search_engines_with_judges_library.ipynb

```diff
@@ -0,0 +1,1687 @@
+{
```

Comment:
<p>Remove the last sentence</p><p> </p><p>"...reading in our <a href="https://huggingface.co/datasets/quotientai/natural-qa-random-67-with-AI-search-answers/tree/main/data" target="_blank">data</a> that now contains the search results."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Done!</p>